wadas/domain/database.py

```diff
@@ -112,47 +124,92 @@ def destroy_instance(cls):
     def create_session():
         """Method to create a session to perform operation with the db"""
 
-        engine = DataBase.get_engine()
-        if engine:
+        if engine := DataBase.get_engine():
```

Comment:
It's better to convert this to classmethod, and use cls.get_engine() instead of directly accessing the class with its name.